meilisearch/index.py

```diff
@@ -249,7 +249,7 @@ def search(self, query: str, opt_params: Optional[Dict[str, Any]] = None) -> Dic
             body=body
         )
 
-    def get_document(self, document_id: str) -> Dict[str, Any]:
+    def get_document(self, document_id: str, fields: Optional[List[str]] = None) -> Dict[str, Any]:
```

Comment:
Do you prefer adding fields as a parameter of the function or a parameter object like on the other routes in case tomorrow we add new params on `get_document` ? 

ex:
```
    def get_document(self, document_id: str,  parameters: Optional[Dict[str, Any]] = None) -> List[Dict[str, Any]]:
```

With that we can add new parameters for the `get_document` route without having to create new function parameters each time.

Reply:
I asked the product team and it is quite likely! I will modify it